docs/reference/settings.md

```diff
@@ -292,7 +292,7 @@ This setting lets you provide your own document model for use in Wagtail, which
 WAGTAILDOCS_DOCUMENT_FORM_BASE = 'myapp.forms.MyDocumentBaseForm'
 ```
 
-This setting lets you provide your own Document base form for use in Wagtail, which might extend the built-in `BaseDocumentForm` class or replace it entirely.
+This setting lets you provide your own Document base form for use in Wagtail, which should extend the built-in `BaseDocumentForm` class.
```

Comment:
Would `must` be the more appropriate term here?  `should` implies recommended but not required.